policy/lib/include_exclude_rules.rego

```diff
@@ -0,0 +1,56 @@
+package lib
+
+#-----------------------------------------------------------------------------
+# Used in main_denies.rego to determine if a package should be skipped.
+# (Somewhat duplicates the non_blocking_checks behavior and hopefully will
+# replace that entirely in future.)
+#
+
+package_included(package_name) {
+	any_included_in(with_wildcard(package_name), include_rules)
+	none_included_in(with_wildcard(package_name), exclude_rules)
+}
+
+package_excluded(package_name) {
+	any_included_in(with_wildcard(package_name), exclude_rules)
+}
+
+package_excluded(package_name) {
+	none_included_in(with_wildcard(package_name), include_rules)
+}
+
+with_wildcard(package_name) := {package_name, "*"}
```

Comment:
I see that this works as intended but I don't quite understand how. If we are always adding a wildcard to the set, wouldn't the matching always be true?

Reply:
Ah right. Looking at this with fresh eyes makes sense. I don't think it needs to be separated for clarity sake.